src/hypertrace/agent/config/agent-config.yaml

```diff
@@ -0,0 +1,29 @@
+service_name: pythonagent_001
+reporting:
+  endpoint: http://localhost:9411/api/v2/spans1
+  secure: true
+  trace_reporter_type: ZIPKIN
+  token: "TestToken"
+  opa:
+    endpoint: "https://opa.traceableai:8181/"
+    poll_period_seconds: 50
+    enabled: true
+data_capture:
+  http_headers:
+    request: false
+    response: false
+  http_body:
+    request: false
+    response: false
+  rpc_metadata:
+    request: false
+    response: false
+  rpc_body:
+    request: false
+    response: false
+  body_max_size_bytes: 123457
+propagation_formats: "B3"
```

Comment:
propagation format is a list, e.g. `["B3"]`

Reply:
As discussed in slack, that fix is in another PR that will be applied after this one.